### DISCUSSION TITLE
How Create Multiple Pages on Web Server After Login Wifi Succesfuly

### DISCUSSION THREAD

**Discussion by ozonerik** [ESP32-Web/WiFi-Manager_Library#5] [Q&A] [answered]
How Create Multiple Pages on Web Server After Login Wifi Succesfuly, like
https://esp32io.com/tutorials/esp32-web-server-multiple-pages

**Comment by FarazFKhan** [comment, marked as answer]
Hello, Thank you for opening this discussion. There are currently no plans to have multiple pages accessible via different url endings. We do have plans on having more like an app like experience where you can navigate between a few main pages without the url changing. But this will take quite some time to release.

**Reply by ozonerik** [reply]
oke, thanks

**Comment by MuzammilPatel** [comment]
Hi @FarazFKhan, Just wondering do you have any plans to implement multiple pages or adding more raido/drop down buttons in upcoming releases?

I was basically interested in adding more user provided option available along side selecting Wifi SSid and password, 
For eg, a drop down menu for selecting counrty/region.

**Reply by FarazFKhan** [reply]
Hello Muzammil, 

There are plans to add various input methods to control & monitor different electronic projects. Probably going to be released in v2.0 by end of 2025.

No plans currently to further add functionality to the WiFi setup page though.
